docs/_includes/date-picker.html

```diff
@@ -1 +1,282 @@
-<!-- TODO: add this before merging; HTML structure is very large so it probably shouldn't be on the main skin site -->
\ No newline at end of file
+<div id="date-picker">
+    {% include section-header.html name="date-picker" version=page.versions.date-picker %}
+
+    <p>The date picker component is used to display a calendar from which dates can be selected, either as single items or as a range.</p>
+
+    <h3>Single Pane</h3>
+
+    <button class="btn" onclick="document.getElementById('date-picker-example').toggleAttribute('hidden')">Show/Hide</button>
+
+    <div id="date-picker-example" class="date-picker">
```

Comment:
Agreed. Ideally we should just include the markup for lightbox dialog and have the datepicker pop open in that. We do that in other places, like Infotip

Reply:
Hmm, not sure if we need dialog here. Was initially thinking it would just be like other simple button flyouts. Not _totally_ averse to dialog, needs more thought, but If we do it will be `aria-modal=false`.

Reply:
I think "Choose Date" or "Choose Dates" would be better as a label for the dialog. Date picker is not a term that users might be familiar with.